recipes-overlayed/busybox/busybox_1.%.bbappend

```diff
@@ -1,2 +1,2 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-SRC_URI_append_rpb = " file://enable-setsid-tty.cfg file://enable-install.cfg"
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+SRC_URI:append_rpb = " file://enable-setsid-tty.cfg file://enable-install.cfg"
```

Comment:
you need :append:rpb here (and perhaps in other places too)

Reply:
True